symplyphysics/laws/optics/refraction_angle_from_environments.py

```diff
@@ -0,0 +1,39 @@
+from symplyphysics import (
+    symbols, Eq, pretty, solve, units, Quantity, sin,
+    validate_input, validate_output, expr_to_quantity
+)
+
+from sympy.physics.units.definitions.dimension_definitions import angle as angle_type
+
+# Description
+## If ray of light comes from one media to another, it refracts.
+## Incoming ray, refracted ray and perpendicular to media boundary are within the same flat.
+
+## Law: n1 * sin(alpha) = n2 * sin(beta)
+## Where:
+## n1 is refractive index of first media,
+## n2 is refractive index of second media,
+## alfa is incoming angle,
+## beta is refraction angle.
+
+# Conditions
+## For the sake of sanity alfa and beta are from 0 to pi/2.
+
+refractive_index_1 = symbols('refractive_index_1')
+refractive_index_2 = symbols('refractive_index_2')
```

Comment:
Maybe come up with more meaningful names? I suggest `incidence_refraction_index` and `resulting_refraction_index`. For angles: `incidence_angle` and `refraction_angle`